tests/recipes/test_wind_recipes.py

```diff
@@ -0,0 +1,110 @@
+from copy import deepcopy
+
+from tests.recipes.base_recipe import BaseRecipes
+from tests.smrf_test_case import SMRFTestCaseLakes, SMRFTestCase
+
+
+class TestWindNinjaRecipes(SMRFTestCaseLakes, BaseRecipes):
+    """Test the wind ninja recipes
+    """
+
+    def test_wind_ninja_recipe(self):
+        """Test the wind ninja recipe
+        """
+
+        # no changes to the base config which is a
+        # wind ninja run in the Lakes
+        config = deepcopy(self.base_config)
```

Comment:
Why the copy here?
The `master_config` method returns a dict based of the `base_config`, which will copy values and not value references from the `basic_config`

Reply:
You are correct, no reason for it here. In other tests where we modify the config file, this was helping to not change the base instance.